systemd.md

```diff
@@ -1,10 +1,15 @@
 # Manual Linux setup with systemd
 
+Note: this is usually only necessary in case your distribution doesn't already
+provide a yubikey-agent as a package.
+
+Refer to [the README](README) for a list of distributions providing packages.
+
 First, install Go and the [`piv-go` dependencies](https://github.com/go-piv/piv-go#installation), build `yubikey-agent` and place it in `$PATH`.
 
 ```text
 $ git clone https://filippo.io/yubikey-agent && cd yubikey-agent
-$ go build && sudo cp yubikey-agent /usr/local/bin/
+$ go build && sudo cp yubikey-agent /usr/bin/
```

Comment:
This being a manual installation instruction, `/usr/local` feels more appropriate?

Reply:
Depends on your definition of manual.
There's no need for `yubikey-agent` to be in a global PATH like `/usr/[local/]bin/` - it could very well just live somewhere in `$src/yubikey-agent`, somewhere else in the home directory.

I explained my reasoning in https://github.com/FiloSottile/yubikey-agent/pull/43/commits/aa688918a39d00725a4ca8c3a01bb99a6520d8c6:

> Also, update the desired install location to /usr/bin. At least for
> distro-installed unit files, it's the usual locations, and telling users
> to manually update the unit file after copying probably is more
> annoying than just asking to install into `/usr/bin` in first place.

We could also not specify any absolute path in `ExecStart`, assuming we only run on systemd >= 239: https://github.com/systemd/systemd/commit/5008da1ec1cf2cf8c15b702c4052e3a49583095d

My main goal here is to make this unit file usable as-is for most modern distros, and `/usr/bin` is the location binaries are usually installed into.

What's your preference?